add_map_plugin.py

```diff
@@ -0,0 +1,41 @@
+import sys
```

Comment:
Does this script really belong at the root directory of this repo?

Reply:
Actually, it looks like the existing scripts under the `script/` directory is obsolete. Let's remove this and place this script there. Update the documentation accordingly, if applicable.

Reply:
Yep it is slightly misplaced.. will relocate it